about.html

```diff
@@ -0,0 +1,220 @@
+          <div id="wrap" class="wrap">
+            <div id="bar1" class="menu"></div>
+            <div id="bar2" class="menu"></div>
+            <div id="bar3" class="menu"></div>
+          </div>
+        </div>
+
+        <div id="navlistdisplay2" class="row">
+          <ul class="nav-list-mobile">
+            <li><a href="./index.html"> Home</a></li>
+            <li>
+              <a id="logoiconabout" href="./about.html"> About Us</a>
+            </li>
+            <li>
+              <a id="pastbatchiconabout" href="#past-Batch-section">
+                Info</a
+              >
+            </li>
+            <li>
+              <a id="partnericonabout" href="#partner-section">Our Partners</a>
+            </li>
+            <li><a href="./index.html">Latest</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="container about-home-container">
+        <div class="row about-heading-1">
+          <h1>Welcome!</h1>
+        </div>
+        <div class="row about-heading-2">
+          <h2>
+            AFRICAN DANCE CONFERENCE 2023
+          </h2>
+        </div>
+        <div class="row about-para-heading">
+          <p>
+            Come enjoy a joyful harmonious infusion of cultural histories and stories fro more than 50 nations and more than 100 tribal commuNities and languages.
+          </p>
+        </div>
+        <div class="row about-heading-3">
+          <h3>
+            If you have any question about the conference, please feel free to contact us
+           
+          </h3>
+        </div>
+        <div class="row about-heading-4">
+          <h4>info@afrivandanceconference.com</h4>
+        </div>
+      </div>
+    </section>
+
+    
+    <!-- Past Batch section start from here -->
+    <section id="past-Batch-section" class="contianer-fluid past-section">
+      <div class="container">
+        <div class="row">
+          <div class="col-12 d-flex justify-content-center past-text">
+            See the Past conferences
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center">
+          <div class="col-1 past-text-line">line</div>
+        </div>
+        <div class="row past-para-heading">
+          <p>
+            Take a look of the past conferences which took place in 2021 and 2022.
+          </p>
+        </div>
+        <div class="past-pic">
+          <div class="pastimage1">
+            <h2>2021</h2>
+            <p>Award Cermony in California</p>
+          </div>
+          <div class="pastimage2">
+            <h2>2022</h2>
+            <p>Award Cermony in America</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Partner section start from here -->
+
+    <section id="partner-section" class="container-fluid partner-section2">
+      <div class="container partner-program-container ">
+          <div class="row">
+              <div class="col-12 d-flex justify-content-center partner-main-text">Partners</div>
```

Comment:
- [ ] Please make sure your about-page sections don't overlap when resized. 

**Your about page**
![image](https://user-images.githubusercontent.com/74411135/235281228-e53945d2-2201-40db-a720-dfca023218ee.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪